components/workExperience/SquareWorkExperienceCard.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
```

Comment:
Not necessary here..

Reply:
![Screenshot 2023-09-13 at 2 07 23 AM](https://github.com/DevTaehong/portfolio_jsm/assets/71358207/8a60b160-152b-4a43-835f-ff5f3f23bb1c)
When I removed that, I got this error. I think it's related to NextUI components